plugins/modules/database/postgresql/postgresql_privs.py

```diff
@@ -864,7 +864,7 @@ def add_grant_option(self):
                 self.query[-1] += ' WITH ADMIN OPTION;'
             else:
                 self.query[-1] += ' WITH GRANT OPTION;'
-        else:
+        elif self._grant_option is not None:
             self.query[-1] += ';'
             if self._obj_type == 'group':
                 self.query.append('REVOKE ADMIN OPTION FOR {0} FROM {1};'.format(self._set_what, self._for_whom))
```

Comment:
```suggestion
        elif self._grant_option is not None:
            self.query[-1] += ';'
            if self._obj_type == 'group':
                self.query.append('REVOKE ADMIN OPTION FOR {0} FROM {1};'.format(self._set_what, self._for_whom))
```
after this condition add
```
else:
    self.query[-1] += ';'
```
the module combines a long query from several other queries separated by `;`

Reply:
Cool, thanks for checking it for me locally.
BTW what do you think about changing ```elif self._grant_option is not None:``` with ```elif self._grant_option is False:``` it might be more intuitive

Reply:
I thought of that, sounds good